src/explore-education-statistics-common/src/modules/find-statistics/components/charts/MapBlock.tsx

```diff
@@ -1,88 +1,680 @@
+                  indicator => meta.indicators[indicator],
+                )}
+              />
+            </div>
+
+            <div className="govuk-form-group govuk-!-margin-bottom-6">
+              <FormSelect
+                name="selectedLocation"
+                id="selectedLocation"
+                label="Select a location"
+                value={selected.location}
+                onChange={e => this.selectLocationOption(e.currentTarget.value)}
+                options={options.location}
+              />
+            </div>
+
+            {groupOptions.map(grouping => (
+              <div
+                key={grouping.id}
+                className="govuk-form-group govuk-!-margin-bottom-6"
+              >
+                <FormSelect
+                  name="grouping"
+                  id={grouping.id}
+                  label={grouping.title}
+                  value={grouping.selected}
+                  onChange={e =>
+                    this.updateSelectedGroupOption(
+                      grouping,
+                      e.currentTarget.value,
+                    )
+                  }
+                  options={grouping.options}
+                />
+              </div>
+            ))}
+          </form>
+
+          {selected.results.length > 0 ? (
+            <div>
+              {selected.results.map(result => (
+                <div
+                  key={result.id}
+                  className="dfe-dash-tiles__tile govuk-!-margin-bottom-6"
+                >
+                  <h3 className="govuk-heading-m dfe-dash-tiles__heading">
+                    {meta.indicators[result.id].label}
+                  </h3>
+                  <p
+                    className="govuk-heading-xl govuk-!-margin-bottom-2"
+                    aria-label={meta.indicators[result.id].label}
+                  >
+                    <span>
+                      {' '}
+                      {result.value}
+                      {meta.indicators[result.id].unit}{' '}
+                    </span>
+                  </p>
+                  <Details
+                    summary={`What is ${meta.indicators[result.id].label}?`}
+                  >
+                    Description for {meta.indicators[result.id].label}
+                  </Details>
+                </div>
+              ))}
+            </div>
+          ) : (
+            ''
+          )}
+
+          {selected.indicator !== '' ? (
+            <div>
+              <h3 className="govuk-heading-s">
+                Key to {meta.indicators[selected.indicator].label}
+              </h3>
+              <dl className="govuk-list">
+                {legend &&
+                  legend.map(({ min, max, idx }) => (
+                    <dd className={styles.legend} key={idx}>
+                      <span className={styles[`rate${idx}`]}>&nbsp;</span> {min}
+                      {meta.indicators[selected.indicator].unit}&nbsp; to {max}
```

Comment:
Similar to above, we could replace these JSX nodes with a single templated string